bugbug/tools/code_review.py

```diff
@@ -86,21 +98,28 @@ class HunkNotInPatchError(ModelResultError):
 {comments}"""
 
 
-PROMPT_TEMPLATE_FILTERING_ANALYSIS = """Please, double check the code review provided for the patch below.
+PROMPT_TEMPLATE_FILTERING_ANALYSIS = (
+    """Please, double check the code review provided for the patch below.
 Just report the comments that are:
 - applicable for the patch;
-- consistent with the source code in Firefox;
-- focusing on reporting possible bugs, major readability regressions, or similar concerns.
+- consistent with the source code"""
+    + (" in Firefox" if COND_MENTION_MOZILLA else "")
+    + """;
+- focusing on reporting possible bugs, major readability regressions, or similar concerns;
+- filter out any descriptive comments;
+- filter out any praising comments;
+- filter out any comments that do not suggest code changes.
```

Comment:
This last one might be too strict, have you tested it?
We might want to test it on our data to make sure it is not filtering out too much.

Reply:
Actually on the few examples I tested it's mostly lines 109-110 and the example line 137 that had an impact, we could remove line 111

Reply:
Let's remove it for now then